oak_utils/src/lib.rs

```diff
@@ -204,3 +204,42 @@ where
         .compile_protos(inputs, includes)
         .expect("could not run prost-build");
 }
+
+/// Options for building gRPC code.
+pub struct CodegenOptions {
+    /// Specify whether to build client related code.
+    pub build_client: bool,
+    /// Specify whether to build server related code.
+    pub build_server: bool,
+}
+
+/// Generate gRPC code from Protobuf using `tonic` library.
+pub fn generate_grpc_code(
```

Comment:
nit: `prost_build` and `tonic_build` both put this method on the config struct itself, which might make this a more fluent API and potentially be more familiar to users. This is probably personal preference though, what you have is also fine 😄